scripts/RunHPCNodes.py

```diff
@@ -0,0 +1,115 @@
+# Taco Niet
+# Date:  September 26, 2016 - 2018
+# Script to run OSeMOSYS with various parameters using full nodes on a SLURM scheduler managed HPC system.
+
+#!/usr/bin/python
+import subprocess # for running glpsol
+from shutil import copy2 # Allow us to make the needed copies of the parameter file
+from math import sqrt
+import zipfile, os, time, io
+
+if __name__ == '__main__':
+    modelruns = []
+    tailcommands = []
+
+	# Setup task list here.  This can be created by a series of nested for loops.
+    param3List = ['10.6', '15.9', '21.2']
+    param2List = ['0000', '0050', '0100']
+    param1List = ['35', '65', '100']    
+
+    NumRunsTotal = len(param2s)*len(param3Levels)*len(param1s)
```

Comment:
Should be `param1List`, `param2List`, `param3List`?

Reply:
Updated.  Thanks!